site/docs/install.md

```diff
@@ -7,9 +7,9 @@ title: Installing Bazel
 
 See the instructions for installing Bazel on:
 
-*   [Ubuntu Linux (16.04, 15.10, and 14.04)](install-ubuntu.md)
-*   [Mac OS X](install-os-x.md)
-*   [Windows (experimental)](install-windows.md)
+*   [Linux](install-ubuntu.md)
```

Comment:
Ok I have read a bit more from the website, we actually should keep ubuntu linux here (with the version), revert the change on the ubuntu page, then maybe add the line "For other platforms, ..." something like (supported platform are CentOS 6.x and 7.x and Fedora 22).